### PR TITLE
Documentation and release note changes for Astronomer CLI 0.27.2

### PR DESCRIPTION
**Pull request by jwitz**
This PR includes:

- Release notes for 0.27.2
- A new document about importing IDP groups as teams
- Doc changes related to improvements in the CLI

### CODE REVIEW

File: enterprise/cli-reference.md [hunk 1]
```diff
@@ -559,7 +559,7 @@ Shows logs for the Scheduler or Webserver in your local Airflow environment.
 
 ### Usage
 
-Run `astro dev logs [flags]` to start tracking logs for your Scheduler or Webserver in your CLI terminal window.
+Run `astro dev logs [flags]` to start tracking logs for your Scheduler, Webserver, or Triggerer in your CLI terminal window.
```
Comment (neel-astro) [inline enterprise/cli-reference.md:562]: Should we also mention the `Triggerer` container in `astro dev start` section as well?

File: enterprise/release-notes.md [hunk 2]
```diff
@@ -15,6 +15,53 @@ Astronomer v0.27 is the latest Stable version of Astronomer Enterprise, while v0
...
+:::danger Breaking Change
+
+The latest version of the Astronomer CLI uses Docker engine `1.13.1` to run Airflow locally via `astro dev`. If you haven't done so already, ensure that your version of Docker Engine is at least `1.13.1` before upgrading the Astronomer CLI to v0.27.2. If your Docker Engine version is `<1.13.1`, then `astro dev` commands will not work on your local machine.
+
+:::
+
+Astronomer CLI v0.27.2 includes several improvements to the local development experience:
+
+- You no longer need to turn off Docker Buildkit to run `astro dev start`.
+- You can now run the Triggerer in a local Airflow environment, meaning that you can test DAGs that include [deferrable operators](https://airflow.apache.org/docs/apache-airflow/stable/concepts/deferring.html) before pushing them to a Deployment. Triggerer logs appear alongside Webserver and Scheduler logs when you run `astro dev logs`.
```
Comment (neel-astro) [inline enterprise/release-notes.md:33]: Should we also mention that `Triggerer` container will only spawn when airflow version is `>= 2.2.0`

**Review by neel-astro — COMMENTED** [review]
Have verified changes related to CLI release and they mostly looks good, have added few minor comments

File: enterprise/release-notes.md [hunk 3]
```diff
@@ -15,6 +15,53 @@ Astronomer v0.27 is the latest Stable version of Astronomer Enterprise, while v0
...
+
+:::danger Breaking Change
+
+The latest version of the Astronomer CLI uses Docker engine `1.13.1` to run Airflow locally via `astro dev`. If you haven't done so already, ensure that your version of Docker Engine is at least `1.13.1` before upgrading the Astronomer CLI to v0.27.2. If your Docker Engine version is `<1.13.1`, then `astro dev` commands will not work on your local machine.
+
+:::
+
+Astronomer CLI v0.27.2 includes several improvements to the local development experience:
+
+- You no longer need to turn off Docker Buildkit to run `astro dev start`.
```
Comment (cmarteepants) [inline enterprise/release-notes.md:32]: I would turn this into a "positive" (I have no idea how to say this in english).
Something similar to: "We now support Docker Buildkit! Even with BuildKit enabled, you are able to run `astro dev start`"

File: enterprise/release-notes.md (hunk 2 above)
Comment (cmarteepants) [reply enterprise/release-notes.md:33]: Agree

**Review by cmarteepants — APPROVED** [review]
Suggestions, but not requirements! Good job!

**Comment by vishwas-astro** [comment]
@jwitz @cmarteepants 

the release that is going to include the IDP groups feature is 0.28.0 and not 0.27.2. 0.28.0 is planned to be released in the first week of February.

We are ready to release 0.27.2 of just our CLI at this time. May be we can restrict this PR to document all the changes being made to the CLI as part of this release.

We can do release notes and feature documentation (such as IDP groups, BYO registry etc) for 0.28.0 in a separate PR.

cc: @neel-astro and @aliotta

File: enterprise/release-notes.md (hunk 2 above)
Comment (jwitz) [reply enterprise/release-notes.md:33]: Does the triggerer run only when a DAG contains a deferrable operator? If so, then we can just focus on the operators and not worry too much about specifying versions. If a user wants to use deferrable operators, they need 2.2.0+ by assumption

**Review by neel-astro — APPROVED** [review]
lgtm 👍

File: enterprise/cli-quickstart.md [hunk 4]
```diff
@@ -21,6 +21,13 @@ If you're an Astronomer Enterprise user, you might use the Astronomer CLI to do
 
 This guide provides steps for installing the CLI, initializing an Astronomer project, and deploying to an Airflow instance on your local machine. For more information on specific CLI workflows and features, read the [Astronomer CLI Reference Guide](cli-reference.md).
 
+## Prerequisites
+
+The Astronomer CLI requires:
+
+- [Docker](https://www.docker.com/) (v18.09 or higher).
+- [Docker Engine](https://docs.docker.com/engine/) (v0.13.1 or higher).
+
```
Comment (paolaperaza) [inline enterprise/cli-quickstart.md:30]: Should we add a note here on Podman? We can still list Docker as the preferred path, but link out to https://docs.astronomer.io/enterprise/cli-podman

File: enterprise/cli-reference.md [hunk 5]
```diff
@@ -591,11 +592,12 @@ Runs a single [Airflow CLI command](https://airflow.apache.org/docs/apache-airfl
 
 ## astro dev start
 
-Initializes a local Airflow environment on your machine by creating a Docker container for each of Airflow's 3 core components:
+Initializes a local Airflow environment on your machine by creating a Docker container for each of Airflow's core components:
 
 - Postgres
 - Scheduler
 - Webserver
+- Triggerer (only when using deferrable operators)
```
Comment (paolaperaza) [inline enterprise/cli-reference.md:600]:
```suggestion
Initializes a local Airflow environment on your machine by creating a Docker container for each of Airflow's core components:

- Postgres
- Scheduler
- Webserver
- Triggerer

:::info

Logs for the Airflow Triggerer will only show for local environments using Deferrable Operators and running Astronomer Certified 2.0+.

:::
```

File: enterprise/release-notes.md [hunk 6]
```diff
@@ -15,6 +15,45 @@ Astronomer v0.27 is the latest Stable version of Astronomer Enterprise, while v0
...
+
+:::danger Breaking Change
+
+The latest version of the Astronomer CLI uses Docker engine `1.13.1` to run Airflow locally via `astro dev`. If you haven't done so already, ensure that your version of Docker Engine is at least `1.13.1` before upgrading the Astronomer CLI to v0.27.2. If your Docker Engine version is `<1.13.1`, then `astro dev` commands will not work on your local machine.
+
+:::
+
+Astronomer CLI v0.27.2 includes several improvements to the local development experience:
+
+- You can now run `astro dev start` with Docker Buildkit enabled.
```
Comment (paolaperaza) [inline enterprise/release-notes.md:32]:
```suggestion
- You can now run `astro dev start` with Docker Buildkit enabled. This resolves a [common issue](https://forum.astronomer.io/t/buildkit-not-supported-by-daemon-error-command-docker-build-t-airflow-astro-bcb837-airflow-latest-failed-failed-to-execute-cmd-exit-status-1/857) whereby users with Docker Buildkit enabled experienced an error that prevented them from running this command.
```

File: enterprise/release-notes.md [hunk 7]
```diff
@@ -15,6 +15,45 @@ Astronomer v0.27 is the latest Stable version of Astronomer Enterprise, while v0
...
+- You can now run the Triggerer in a local Airflow environment, meaning that you can test DAGs that include [deferrable operators](https://airflow.apache.org/docs/apache-airflow/stable/concepts/deferring.html) before pushing them to a Deployment. Triggerer logs appear alongside Webserver and Scheduler logs when you run `astro dev logs`.
```
Comment (paolaperaza) [inline enterprise/release-notes.md:33]:
```suggestion
- You can now run the Airflow Triggerer in a local Airflow environment. When you run `astro dev start` via the Astronomer CLI, a container for the Triggerer will now be started alongside containers for the Scheduler, Webserver, and Postgres Metadata Database. This means that you can test DAGs that use [deferrable operators](https://airflow.apache.org/docs/apache-airflow/stable/concepts/deferring.html) locally before pushing them to a Deployment on Astronomer. Triggerer logs appear alongside Webserver and Scheduler logs when you run `astro dev logs`. This command will only be successful  in environments running Astronomer Certified 2.2.0+.
```

File: enterprise/release-notes.md [hunk 8]
```diff
@@ -15,6 +15,45 @@ Astronomer v0.27 is the latest Stable version of Astronomer Enterprise, while v0
...
+    triggerer			running		
+    scheduler			running		
+    0.27.2_a64c1a-postgres-1	running		5432
+    ```
+
+    To change the default names of these containers, run `astro config set <airflow-component>.container_name <new-component-container-name>`.
+
+### Additional Improvements
+
+- The Astronomer CLI can now be installed on machines with an [Apple M1 chip](https://www.apple.com/newsroom/2020/11/apple-unleashes-m1/) via both curl and HomeBrew.
```
Comment (paolaperaza) [inline enterprise/release-notes.md:50]: Link to CLI install? Also `HomeBrew` > `Homebrew`?

**Review by paolaperaza — COMMENTED** [review]
A few comments here @jwitz !

File: enterprise/release-notes.md (hunk 7 above)
Comment (jwitz) [reply enterprise/release-notes.md:33]: I cut this down a bit

**Comment by jwitz** [comment]
@neel-astro Informed me that theCLI would be releasing without a platform 0.27.2 release. I think the only way to make this clear in release notes is by starting a doc specifically for CLI release notes. We don't want to give people the impression that they can start an upgrade process on Enterprise, only to find that 0.27.2 doesn't exist.

**Review by vishwas-astro — APPROVED** [review]

**Pull request merged**